src/uwtools/tests/cli/test_validate_config.py

```diff
@@ -21,7 +21,7 @@
 def files(tmp_path):
     cfgfile = tmp_path / "cfg.yaml"
     logfile = tmp_path / "log"
-    schemafile = fixture_path("schema_test_good.yaml")
+    schemafile = fixture_path("/dev/null")
```

Comment:
`/dev/null` always exists, and since the CLI tests only need to confirm existence, and file will do. I think it's misleading to supply a real file with a real schema that is never actually used.